libbeat/docs/dashboards.asciidoc

```diff
@@ -12,57 +12,45 @@
 
 
 [[load-kibana-dashboards]]
-==== Importing the Dashboards
+==== Set up the dashboards
 
-{beatname_uc} comes packaged with the `scripts/import_dashboards` script that you can use to import the example dashboards,
-visualizations, and searches for {beatname_uc}. The script also creates an index pattern,
-+{beatname_lc}-*+, for {beatname_uc}.
+{beatname_uc} comes packaged with example Kibana dashboards, visualizations,
+and searches for visualizing {beatname_uc} data in Kibana. Before you can use
+the dashboards, you need to run the `setup` command to create the index
+pattern, +{beatname_lc}-*+, and load the dashboards into the Kibana index.
 
-The steps in this section show how to import {beatname_uc} dashboards. You may want to import dashboards for more than
-one Beat or specify import options that aren't described here. See {beatsdevguide}/import-dashboards.html[Importing Existing Beat Dashboards]
-in the _Beats Platform Reference_ for a full list of command-line options.
+NOTE: Starting with Beats 6.0.0, the dashboards are loaded via the Kibana API.
+This requires a Kibana endpoint configuration. You should have configured the
+endpoint in <<{beatname_lc}-configuration>>. If you did't, configure it now
+and then restart {beatname_uc} before proceeding.
 
-To import the Kibana dashboards for {beatname_uc}:
+//I'm just assuming that a restart is required.
+
+To set up the Kibana dashboards for {beatname_uc}:
 
 ifdef::allplatforms[]
 
 *deb, rpm, and mac:*
 
-From the directory where you installed {beatname_uc}, run the `import_dashboards` script.
+From the directory where you installed {beatname_uc}, run:
+
+//TODO: I'm just guessing at these commands. I need to see how they really work.
 
 ["source","sh",subs="attributes,callouts"]
 ----------------------------------------------------------------------
-./scripts/import_dashboards
+./{beatname_lc} setup
 ----------------------------------------------------------------------
 
+ifeval::["{beatname_lc}"!="auditbeat"]
+
 *docker:*
 
 ["source","sh",subs="attributes"]
 ----------------------------------------------------------------------
-docker run {dockerimage} ./scripts/import_dashboards
+docker run {dockerimage} ./{beatname_lc} setup
```

Comment:
We'll need to test this, but I think the command only needs to be `setup` (without the ./auditbeat).